skymp5-functions-lib/index.ts

```diff
@@ -150,11 +150,16 @@ export const isDead = (mp: Mp, self: PapyrusObject, args: PapyrusValue[]): boole
   return mp.get(selfId, 'isDead');
 }
 
-export const spLog = (mp: Mp, self: null, args: PapyrusValue[]): PapyrusObject | undefined => {
-  var str = 'From papyrus: ';
+const printArgs = (args: PapyrusValue[]): string => {
+  var out = '';
   for (const pVal of args) {
-    str += ' ' + ((typeof pVal === 'object' || Array.isArray(pVal)) ? JSON.stringify(pVal) : pVal) + ';';
+    out += ' ' + ((typeof pVal === 'object' || Array.isArray(pVal)) ? JSON.stringify(pVal) : pVal) + ';';
   }
+  return out;
```

Comment:
btw would be nice if there was no space in the beginning and no `;` in the end

Reply:
Just followed code that was already there

Reply:
`; ` is used as delimiter for readability.